presto-main/src/main/java/com/facebook/presto/operator/scalar/annotations/ParametricScalarImplementation.java

```diff
@@ -629,6 +629,17 @@ else if (Stream.of(annotations).anyMatch(BlockPosition.class::isInstance)) {
                     }
                 }
             }
+            long valueArgumentsReturnNullOnNullCount = argumentProperties.stream()
```

Comment:
nit: Is it actually  counting *not* return null on null ? :) -- seems not matching the variable name ?  :)